orchagent/swssnet.h

```diff
@@ -76,6 +77,53 @@ inline static sai_ip_prefix_t& copy(sai_ip_prefix_t& dst, const IpAddress& src)
     return dst;
 }
 
+static int getPrefixLenFromAddr(const uint8_t *addr, int len)
```

Comment:
Not sure I understand this. What is the change for? If I assume correct, please use` family=Ipv4?32:128`

Reply:
I suppose we need this, e.g mask 255.255.0.0 should be convert to 16.

Reply:
ok. Then I suggest rename to `getPrefixLenFromAddrMask`